### PR TITLE
Combine then statements into one step

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
Make combined steps in places where acceptance tests are "forced" to have repeated when-then-when-then sections
Related issue: #33374 

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/apiMain/checksums.feature [hunk 1]
```diff
@@ -257,7 +254,7 @@ Feature: checksums
     Given using <dav_version> DAV path
     When user "user0" uploads file "data/textfile.txt" to "/textfile0.txt" with checksum "MD5:d70b40f177b14b470d1756a3c12b963a" using the WebDAV API
     And user "user0" requests the checksum of "/textfile0.txt" via propfind
```
[the remark this answers is not in the document]
Comment (paurakhsharma) [reply tests/acceptance/features/apiMain/checksums.feature]: Good, catch ;)

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -734,7 +734,37 @@ public function theBackgroundJobsModeShouldBe($mode) {
...
+	public function theLogLevelShouldBe($logLevel) {
+		$this->featureContext->invokingTheCommand(
+			"config:system:get loglevel"
+		);
+		$lastOutput = $this->featureContext->getStdOutOfOccCommand();
+		PHPUnit_Framework_Assert::assertEquals($logLevel, \trim($lastOutput));
+	}
+
+	/**
+	 * @When the administrator adds a config key :key with value :value in app :app using the occ command
```
[the remark this answers is not in the document]
Comment (paurakhsharma) [reply tests/acceptance/features/bootstrap/OccContext.php]: I might have made mistake while resolving the conflict.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@paurakhsharma drone CI failed. Please look. It will be good to have these changes in core tonight.

**Comment by individual-it** [comment]
backport in #33501

**Pull request merged**